test/acceptance/tests.py

```diff
@@ -389,64 +386,6 @@ def test_peer_assessment(self):
         self.do_peer_assessment()
 
 
-class PeerAssessmentTestStaffOverride(OpenAssessmentTest):
```

Comment:
I think it is. ORA is so massively complicated that I can imagine a bug that only affects problems with only one step in them that won't be caught by tests that have problems with two or more steps.